packages/YouTubePlayer/src/YouTubePlayer.unit.mjs

```diff
@@ -2,10 +2,26 @@ import { dirname } from 'path';
 import { fileURLToPath } from 'url';
 import test from 'ava';
 import getDOM from '../../../src/testHelpers/getDOM.mjs';
+import jsdom from 'jsdom';
```

Comment:
Ist das ein npm Päckli? Sollte dann nicht auch das package.json geupdated werden?

Reply:
Jap; es kommt in package.json bereits vor, weil wir's auch früher zum Testen verwendet haben, aber via Import in getDOM.